### PR TITLE
Amp analytics: Timer spec doc

### PR DESCRIPTION
**Pull request by pomeroyr**
Adds public documentation for starting and stopping timers.

### CODE REVIEW

**Comment by pomeroyr** [comment]
@dvoytenko @zhouyx

File: extensions/amp-analytics/amp-analytics.md [hunk 1]
```diff
@@ -441,7 +441,7 @@ Use the scroll trigger (`"on": "scroll"`) to fire a request under certain condit
 
 ##### Timer trigger
 Use the timer trigger (`"on": "timer"`) to fire a request on a regular time interval. Use `timerSpec` to control when this will fire:
-  - `timerSpec` Specification for triggers of type `timer`. The timer will trigger immediately (by default, can be unset) and then at a specified interval thereafter.
+  - `timerSpec` Specification for triggers of type `timer`. The unless a `startSpec` is specified, the timer will trigger immediately (by default, can be unset) and then at a specified interval thereafter.
     - `interval` Length of the timer interval, in seconds.
     - `maxTimerLength` Maximum duration for which the timer will fire, in seconds.
```
Comment (zhouyx) [inline extensions/amp-analytics/amp-analytics.md]: Please note the special case when `stopSpec` and `maxTimerLength` are both present

File: extensions/amp-analytics/amp-analytics.md [hunk 2]
```diff
@@ -459,6 +459,30 @@ Use the timer trigger (`"on": "timer"`) to fire a request on a regular time inte
 }
 ```
 
+To configure a timer which times user events use:
+    - `startSpec` Specification for triggering when a timer starts. Use the value of `on` and `selector` to track specific events.
```
Comment (zhouyx) [inline extensions/amp-analytics/amp-analytics.md]: Direct to the doc on how to specify a trigger event. Please also add documentation here that nested timer events are not supported.

File: extensions/amp-analytics/amp-analytics.md (hunk 1 above)
Comment (pomeroyr) [reply extensions/amp-analytics/amp-analytics.md]: Done.

File: extensions/amp-analytics/amp-analytics.md (hunk 2 above)
Comment (pomeroyr) [reply extensions/amp-analytics/amp-analytics.md]: Done.

**Comment by pomeroyr** [comment]
Comments addressed. PTAL @zhouyx

**Comment by aghassemi** [comment]
@zhouyx I am merging this since #11813 hit prod and `startSpec`, etc is live

**Pull request merged**